### ISSUE TITLE
Update example to work with value overhaul

### ISSUE THREAD

**Issue by tsachiherman** [stellar/soroban-examples#217]
### What ?

Update example to work with value overhaul

### Epic

https://github.com/stellar/soroban-tools/issues/471

**Comment by Shaptic** [comment]
@tsachiherman this is normally handled by the Core team (see the [commit history](https://github.com/stellar/soroban-examples/commits/main)); did you mean for this to apply to https://github.com/stellar/soroban-example-dapp/?

**Comment by tsachiherman** [comment]
> @tsachiherman this is normally handled by the Core team (see the [commit history](https://github.com/stellar/soroban-examples/commits/main)); did you mean for this to apply to https://github.com/stellar/soroban-example-dapp/?

you're correct @Shaptic; I meant it to be `soroban-example-dapp`. I'll update that.
